TPL/force-app/main/default/lwc/continuingcareRecordsCase/continuingcareRecordsCase.html

```diff
@@ -1,94 +1,96 @@
 <template>
-    <template if:true={records}>
-           <!-- Continuing Care Records -->
-     <lightning-card title="Continuing Care Records">
-          <!-- Delete Button Code -->
-          <lightning-layout>
-            <lightning-layout-item class="slds-text-align_left">
-                <lightning-record-edit-form object-api-name="Healthcare_Cost__c"  class="slds-text-align_left" id="assignCases" >
-                    <lightning-button label="Delete selected Records" onclick={handleSelect}></lightning-button>
-                </lightning-record-edit-form>
-            </lightning-layout-item>
-        
-            <lightning-layout-item class="slds-text-align_right">
-                <lightning-button label="Refresh" onclick={handleRefresh}></lightning-button>
-            </lightning-layout-item>    
-        </lightning-layout>
+<template if:true={records}>
+        <!-- Continuing Care Records -->
+    <lightning-card title="Continuing Care Records">
+    
 
-         <!-- Handle Filter code -->
-        <lightning-layout>
-            <lightning-layout-item>
-                <lightning-combobox
-                name="group-filter"
-                label="Select records to see" 
-                value={selectedFilter}
-                options={filterOptions}
-                onchange={handleFilterChange}
-                placeholder="Select an option"></lightning-combobox>
-            </lightning-layout-item>   
-        </lightning-layout>
+        <!-- Handle Filter code -->
+    <lightning-layout>
+        <lightning-layout-item>
```

Comment:
These changes are confirmed to be cosmetic in the html definitions to match the other healthcare cost screens and will not alter existing functionality.